app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import os
+
+
+def move_file(commnad: str) -> None:
+    try:
+        command, current_file_name, new_file_path = commnad.split()
+    except ValueError:
+        print("Command must have 3 arguments")
+        return
+    if current_file_name == new_file_path or command != "mv":
```

Comment:
It's better to use a separate condition to check if the command is 'mv' and print an error message if it's not.